### DISCUSSION TITLE
Customer ER in Databricks

### DISCUSSION THREAD

**Discussion by durgesh312** [zinggAI/zingg#889] [Q&A]
Hi, I am trying to perform customer ER in Databricks. The files here are a little complicated to understand. I am using Azure Data Storage. Could help me with the configuration and the files I need to create. I already have traing data available.

**Comment by sonalgoyal** [comment]
Hello @durgesh312 you can use the example notebook under Databricks folder in the main branch. Here is a video demo of how it works https://zinggai.sharepoint.com/:v:/s/Zingg/Ee6UkWvDBERArfcA1VUN194BInNB4OuC8zJFcARZG7kPGQ?e=RiNRfd

hope this helps

**Reply by durgesh312** [reply]
Hello @sonalgoyal thank you for your reply. But unfortunately the link you provided is not working. Please refer to the snapshot.
![Capture](https://github.com/user-attachments/assets/a903e11d-048d-404d-8f20-1d912943d72d)

All the help is much appriciated.

**Reply by suryap-zingg** [reply]
@durgesh312 kindly view the databricks demo at: https://youtu.be/rUAIAP1zJWQ

hope this clarifies.

**Reply by durgesh312** [reply]
@sonalgoyal Thank you so much for the video and the help.
